### PR TITLE
Support local ssd device

### PR DESCRIPTION
**Pull request by andrewgdotcom**

### CODE REVIEW

File: gcloud.conf [hunk 1]
```diff
@@ -43,6 +43,8 @@ ES_DOWNLOAD_URL=$( curl $CURL_ARGS -H "Metadata-Flavor: Google" "http://metadata
 [[ $DEBUG ]] && echo ES_DOWNLOAD_URL=$ES_DOWNLOAD_URL || true
 CUSTOM_ES_JAVA_OPTS=$( curl $CURL_ARGS -H "Metadata-Flavor: Google" "http://metadata.google.internal/computeMetadata/v1/instance/attributes/custom_es_java_opts" )
 [[ $DEBUG ]] && echo CUSTOM_ES_JAVA_OPTS=$CUSTOM_ES_JAVA_OPTS || true
+DATA_DEVICE=$( curl $CURL_ARGS -H "Metadata-Flavor: Google" "http://metadata.google.internal/computeMetadata/v1/instance/attributes/es_data_device" )
+[[ $DEBUG ]] && echo DATA_DEVICE=$DATA_DEVICE || true
```
Comment (itudoben) [inline gcloud.conf:47]: Just for my own info.
curl -H "Metadata-Flavor: Google" http://metadata.google.internal/computeMetadata/v1/instance/attributes/custom_es_java_opts - returns some gibberish  HTML.
What information should this return?

**Review by itudoben — APPROVED** [review]
Good to go!
I just have some personal curiosity :)

File: gcloud.conf (hunk 1 above)
Comment (andrewgdotcom) [reply gcloud.conf:47]: It returns the metadata of the machine that you called it from, based on source IP. The metadata is set in spawner.sh on line 228, and read here. This is how you pass parameters into the individuals slaves.

**Comment by andrewgdotcom** [comment]
#38

**Pull request merged**